src/pages/dashboard/components/dashboardSideMenu/DashboardSideMenu.tsx

```diff
@@ -1,146 +1,113 @@
-              ...applicationLinkBehaviour(history, onExit, ENV.ROUTES.GROUPS, {
-                partyId: party.partyId,
-              }),
-            }
-          : undefined,
-      ],
-    },
-  ];
-  const [selectedItem, setSelectedItem] = React.useState<MenuItem | null>(navigationMenu[0]);
-  const arrayMenu: Array<MenuItem> = navigationMenu.concat(
-    products
-      .filter((p) => p.status === 'ACTIVE')
-      .map((p) => ({
-        groupId: p.id,
-        title: p.title,
-        active: p.authorized ?? false,
-        subMenu: [
-          {
-            groupId: p.id,
-            title: t('overview.sideMenu.product.overview'),
-            active: p.authorized ?? false,
-            onClick: () => invokeProductBo(p, party),
-          },
-          p.userRole === 'ADMIN'
-            ? {
-                groupId: p.id,
-                title: t('overview.sideMenu.product.referents'),
-                active: p.authorized ?? false,
-                ...applicationLinkBehaviour(history, onExit, ENV.ROUTES.PRODUCT_USERS, {
-                  partyId: party.partyId,
-                  productId: p.id,
-                }),
-              }
-            : undefined,
-        ],
-      }))
-  );
 
-  useEffect(
-    () =>
-      setSelectedItem(
-        arrayMenu.find(
-          (m) => m.isSelected || (m.subMenu && m.subMenu.findIndex((m) => m?.isSelected) > -1)
-        ) ?? null
-      ),
-    []
+  const isOVerviewSelected =
+    window.location.pathname ===
+    resolvePathVariables(DASHBOARD_ROUTES.OVERVIEW.path, {
+      partyId: party.partyId,
+    });
+  const isRoleSelected = window.location.pathname.startsWith(
+    resolvePathVariables(ENV.ROUTES.USERS, {
+      partyId: party.partyId,
+    })
+  );
+  const isGroupSelected = window.location.pathname.startsWith(
+    resolvePathVariables(ENV.ROUTES.GROUPS, {
+      partyId: party.partyId,
+    })
   );
-
-  const handleClick = (
-    _event: React.MouseEvent<HTMLDivElement, MouseEvent>,
-    menuItem: MenuItem
-  ) => {
-    setSelectedItem(
-      menuItem.groupId === selectedItem?.groupId && menuItem.subMenu ? null : menuItem
-    );
-    if (menuItem.onClick) {
-      menuItem.onClick();
-    }
-  };
 
   return (
     <Grid container item mt={11}>
       <Grid item xs={12}>
         <List>
-          {arrayMenu &&
-            arrayMenu.map((item) => (
-              <DashboardSideMenuItem
-                key={item.title}
-                color={!item.active ? '#CCD4DC' : 'primary.main'}
-                item={item}
-                selectedItem={selectedItem}
-                handleClick={handleClick}
-              />
-            ))}
+          <DashboardSidenavItem
+            title={t('overview.sideMenu.institutionManagement.overview.title')}
+            handleClick={() =>
+              onExit(() =>
+                history.push(
+                  party.partyId
+                    ? resolvePathVariables(DASHBOARD_ROUTES.OVERVIEW.path, {
```

Comment:
this value is the same as line 41, let's calculate it using the same variable
Same logic also for the other items